wfl/Profiles/RootProfile.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Hypermedia.Affordances;
+using Hypermedia.Transforms;
+using Hypermedia;
+using Symphono.Wfl.Controllers;
+using Symphono.Wfl.Models;
+
+namespace Symphono.Wfl.Profiles
+{
+    public class RootProfile: EntityProfile<Root>
+    {
+        protected override void Configure(EntityConfiguration<Root> configuration)
+        {
+            configuration
+                .UseRepresentationTransform(representation => representation
+                    .WithRepresentation("root")
+                    .WithRepresentation("collection")
+                )
+                .UseLinkTransform(links => links
+                    .WithLink(l => l
+                        .WithRelation("self")
+                        .WithRepresentation("root")
+                        .WithLink<Root, RootController>(
+                            root => rc => rc.Get()
+                        )
+                    )
+               )
+               .UseLinkTransform(links => links
+                    .WithLink(l => l
+                        .WithRelation("restaurants")
+                        .WithRepresentation("restaurant")
```

Comment:
Just out of curiousity, why is the relation different than the representation? Is the representation the expected output type being returned from the link?